app/components/document_list_component.html.erb

```diff
@@ -1,15 +1,15 @@
-<ul class="app-document-list">
+<dl class="app-document-list">
   <% items.each do |item| %>
-    <li class="app-document-list__item">
-      <h3 class="app-document-list__item-title">
+    <div class="app-document-list__item">
```

Comment:
Does this markup pass an HTML validator check?

I don't think a `<dl>` should contain a`<div>`

Reply:
Valid since HTML 5.2 🎉 https://www.stefanjudis.com/today-i-learned/divs-are-valid-elements-inside-of-a-definition-list/